x-pack/plugins/monitoring/public/application/pages/apm/template.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { PageTemplate } from '../page_template';
+import { TabMenuItem, PageTemplateProps } from '../page_template';
+
+interface ApmTemplateProps extends PageTemplateProps {
+  instance?: any;
+}
+
+export const ApmTemplate: React.FC<ApmTemplateProps> = ({ instance, ...props }) => {
+  const tabs: TabMenuItem[] = [];
+
+  if (!instance) {
```

Comment:
For the instance page, we could use the `PageTemplate` directly and we wouldn't need this conditional.

Reply:
I did that with the beats template originally, but then followed @phillipb 's pattern from logstash views instead https://github.com/elastic/kibana/pull/113500/commits/d6fb0b28e19d0adaad1f5adb33cc60dea9175ae5

Not sure I have strong feelings either way, but it'd be nice if all the pages had a consistent tab control model, I think.

Reply:
Opted to drop it here since there's no real logic, just add tabs or don't. Maybe the conditional `tabs.push` makes more sense when you have a few different tab "modes".